xla/service/gpu/ir_emission_utils.cc

```diff
@@ -31,6 +31,7 @@ limitations under the License.
 #include "xla/hlo/ir/hlo_instruction.h"
 #include "xla/hlo/ir/hlo_opcode.h"
 #include "xla/mlir_hlo/mhlo/IR/hlo_ops.h"
+#include "xla/service/gpu/reduction_utils.h"
```

Comment:
We should try to avoid this include here. I recently splitted ir_emission_utils and reduction_utils to be separate, and I would like to keep it that way.

Reply:
ok sure, i think i will need to do a refactoring of the functions here. i will move all the findRealReductionHero related stuff into reduction_utils then